### DISCUSSION TITLE
Unable to get my appropriate output after deployment

### DISCUSSION THREAD

**Discussion by affaansafi** [community/community#122946] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

My portfolio website which I deployed was "https://affaansafi.github.io/portfolio/". I am unable to get my appropriate output after deployment. Please help me .

**Comment by yoannchaudet** [comment]
It's there: https://affaansafi.github.io/portfolio/MY%20PORTFOLIO/.

You probably want to put your content at the root of your site instead of in a `MY PORTFOLIO` folder.

**Comment by LiteBrite82** [comment]
for this deployment, it is best to keep all of your code in the root directory; not in `MY PORTFOLIO`
